### ISSUE TITLE
Support hybrid deployments

### ISSUE THREAD

**Issue by dgageot** [GoogleContainerTools/skaffold#1272]
At every conference, people ask me if Skaffold supports hybrid deployments. When a set of components is too big to fit locally, they want to run some components locally and some remotely.
+ It could be that they want to run a service locally (no container involved) and connect it to a remote cluster.
+ Or they want to run a few components (in k8s) locally connected to a many more components remotely. 

Basically they expect Skaffold to drive something like [telepresence](https://www.telepresence.io/) or [kubefwd](https://github.com/txn2/kubefwd).

**Comment by xeor** [comment]
This would make skaffold much more attractive for us as well.. I'm probably ending up using telepresence or kubefwd, but want something more.. (not sure what I want, but this issue seamed promising 👍)

**Comment by priyawadhwa** [comment]
@xeor, would it be useful for you if skaffold could forward remote services to your local computer via telepresence for you?

**Comment by WoLfulus** [comment]
This is something I'm really interested too. I'm looking for a way to run locally only the projects I'm working on, and yet, make it connect to its dependencies on the cluster (without they being fully exposed on the internet).

Any suggestions on how to make this work? 

@xeor I saw telepresence too, did it work for you?

**Comment by xeor** [comment]
yea. Think I'm going for telepresence.. Haven't really started to use it a lot tho. Looks nice

**Comment by dgageot** [comment]
Let's close this one since there's not a lot of activity on this issue and there's nothing in the short term roadmap
